.github/workflows/prerelease.yml

```diff
@@ -40,6 +40,12 @@ jobs:
           python -Im tox run -e compile-assets,i18n-compile,py$(echo ${{ matrix.python-version }} | tr -d .)-tests-no-cov
           echo "PYTEST_ERRORS=$?" >> $GITHUB_ENV
 
+      - name: "Build and inspect package 📦"
+        uses: hynek/build-and-inspect-python-package@v2
```

Comment:
Does this actually error out? I mean, would this line have caught the missing translation files, and if so, how?

Reply:
Logging, essentially. That makes sense 👍